angular.json

```diff
@@ -71,7 +71,8 @@
 							"-t"
 						],
 						"configDir": ".storybook",
-						"outputDir": "dist/docs/storybook"
+						"outputDir": "dist/docs/storybook",
+            "enableProdMode": false
```

Comment:
Need to set `enableProdMode` to `false` for the story to correctly display. _May_ be addressed by updating to storybook in v8?

 Tested the build in angular 18 and was able to pass navigation extras successfully.